code/_onclick/hud/fullscreen.dm

```diff
@@ -1,4 +1,6 @@
-/mob/proc/overlay_fullscreen(category, type, severity)
+/mob/proc/overlay_fullscreen(category, type, severity, _alpha)
```

Comment:
I don't think you need it to be `_alpha`--just use `alpha`. If `alpha` is used in the body of the proc, then use `src.alpha` there, or even just rename this to `overlay_alpha`.